src/aspire/estimation/covar2d.py

```diff
@@ -1,10 +1,12 @@
 import logging
+import numpy as np
 from scipy.linalg import sqrtm
 from scipy.linalg import solve
 from numpy.linalg import inv
 
+from aspire.utils.BlockDiagonal import BlockDiagonal
```

Comment:
For the consistency with other parts, `BlkDiagMatrix` looks a better class name  compared to `BlockDiagonal`.